### PR TITLE
LIVY-228. Shade json4s dependencies and mitigate py4j difference

### PR DESCRIPTION
**Pull request by jerryshao**

### CODE REVIEW

**Comment by tc0312** [comment]
LGTM.

**Comment by jerryshao** [comment]
\cc @tc0312

File: repl/pom.xml [hunk 1]
```diff
@@ -174,6 +211,15 @@
                         <goals>
                             <goal>copy-dependencies</goal>
                         </goals>
+                        <configuration>
+                            <excludeArtifactIds>
+                                json4s-ast_${scala.binary.version},
+                                json4s-core_${scala.binary.version},
+                                json4s-jackson_${scala.binary.version},
+                                paranamer,
+                                scalap
```
Comment (zjffdu) [inline repl/pom.xml]: why paranamer, scalap is excluded here too ?

Comment (jerryshao) [inline repl/pom.xml]: scalap will be assembled into shade jar, so don't need to copy into jars folder.

**Pull request merged**